index.md

```diff
@@ -120,34 +120,43 @@ A legjobb magyar szerpentin mind közül. Jó útminőség, alacsony-közepes fo
 
-### Bükk / Lillafüred-Mályinka
+### Lillafüred-Mályinka
 
 Rossz útminőségre lehet számítani.
 
 <iframe src="https://www.google.com/maps/embed?pb=!1m24!1m12!1m3!1d42611.9198519277!2d20.555377375210885!3d48.1247986208386!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m9!3e0!4m3!3m2!1d48.104963999999995!2d20.622595!4m3!3m2!1d48.152504699999994!2d20.498784399999998!5e0!3m2!1shu!2shu!4v1589887345567!5m2!1shu!2shu" width="100%" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0" loading="lazy"></iframe>
 
-### Bükk / Mexikóvölgy
+### Mexikóvölgy
 5 pont
 
 Ez egy rally útvonal, így érdemes felkészülni a kissé nehézkesebb közlekedésre. Nagyon keskeny, csak felfelé lehet haladósabban közlekedni, nagyon rossz útminőség. Néha szembeforgalom jön, amikor általában valakinek állóra kell fékezni. Nagyon élvezhető, de nagyon észnél kell lenni, vagy biztosított úton kell menni.
 
 <iframe src="https://www.google.com/maps/embed?pb=!1m24!1m12!1m3!1d30160.11801631419!2d20.681408026723144!3d48.07544540802806!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m9!3e0!4m3!3m2!1d48.0868922!2d20.7149269!4m3!3m2!1d48.0679441!2d20.633659299999998!5e0!3m2!1shu!2shu!4v1589887152544!5m2!1shu!2shu" width="100%" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0" loading="lazy"></iframe>
 
+### Balaton (utolsó frissítés: 2021 szeptember)
+6 pont
+
+Egy rövid, szépen felújított, rövid útvonal köti össze a bükki Balaton települést Borsodnádasdal. Az utat nem rég felújították, így a felülete nagyon jó állapotú, sima. Az útvonal keleti szakaszán, Balatontól a hegytetőig veszettül csúszós fehér aszfalt van, amelyen még egy meleg nyári napon is bőven korcsolyázni lehet. Emiatt érdemes inkább Balaton irányából megcsinálni ezt a szakaszt. A hegytetőtől lefelé megfelelő minőségű és normál tapadású aszfalttal folytatódik az útvonal, és a legvégén két nagyon szűk visszafordítóban végződik. A szakasznak közepes - alacsony forgalma van és egyben kitűnő átvezető szakasznak lehet használni a 2505-ös / 24-es utak és a Hagyony-Zabari illetve Ceredi szerpentinek között.
+
+Délről a Balaton felé vezető útvonal is kifejezetten kellemesen kanyarog, de ott sokkal nagyobb a forgalom is.
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m24!1m12!1m3!1d10656.546836651556!2d20.274454039703873!3d48.10759846590404!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m9!3e0!4m3!3m2!1d48.10715!2d20.30142!4m3!3m2!1d48.1058497!2d20.2649856!5e0!3m2!1shu!2shu!4v1631629380828!5m2!1shu!2shu" width="100%" height="450" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
```

Comment:
iframeket nem néztem, feltételezem h jók

Reply:
miben mas mint az eddigi terkep?

Reply:
Nemtom, a korábbit nem is láttam először. De jó lesz az.